template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/kafka/KafkaDatahubConfigProvider.java

```diff
@@ -36,6 +38,9 @@
         if (KafkaConfigProviderUtils.getCdhVersionForStreaming(source).supportsRangerServiceCreation()) {
             configs.add(config(RANGER_PLUGIN_KAFKA_SERVICE_NAME, GENERATED_RANGER_SERVICE_NAME));
         }
+
+        configs.add(config(KAFKA_DECOMMISSION_HOOK_ENABLED, "true"));
+
```

Comment:
This should only be effective with CDH versions 7.2.12+

You should extend the CdhVersionForStreaming with a 7_2_12 enum and a supportsDecommissionHook() method.

Reply:
Added the requested changes